great_expectations/data_context/store/expectations_store.py

```diff
@@ -244,17 +244,28 @@ def get_key(
         self, suite: ExpectationSuite
     ) -> GXCloudIdentifier | ExpectationSuiteIdentifier:
         """Given an ExpectationSuite, build the correct key for use in the ExpectationsStore."""
+        return self._get_key(name=suite.name, id=suite.ge_cloud_id)
+
+    def get_key_by_name(
+        self, name: str
+    ) -> GXCloudIdentifier | ExpectationSuiteIdentifier:
+        """Given a name, obtain the correct key."""
+        # Workaround to allow callers to obtain a key from name without needing to be
```

Comment:
I think this `workaround` comment should be in the docstring if the method itself is a workaround, but I think a bit more explanation would help. How does `get_key` require needing to aware of the StoreBackend impl?

Reply:
thanks! i removed this method in https://github.com/great-expectations/great_expectations/pull/9139/commits/d9fa2cbbcca96ec608db33cafea99dc3291f8c82